gemcalibration/include/gem/calib/Calibration.h

```diff
@@ -0,0 +1,240 @@
+/** @file Calibration.h */
+
+#ifndef GEM_GEMCAL_CALIBRATION_H
+#define GEM_GEMCAL_CALIBRATION_H
+
+#include <string>
+#include <vector>
+
+#include "gem/base/GEMApplication.h"
+#include "gem/utils/Lock.h"
+#include "gem/utils/LockGuard.h"
+
+//#include "gem/daqmon/exception/Exception.h"
+#include "gem/utils/exception/Exception.h"
+//#include "gem/daqmon/DaqMonitor.h"
+
+#define NAMC 12
+#define NSHELF 2
+
+
+namespace gem {
+  namespace calib {
+
+    enum calType {NDEF, GBTPHASE, LATENCY, SCURVE, SBITARMDACSCAN, ARMDACSCAN, TRIMDAC, DACSCANV3, CALIBRATEARMDAC}; 
```

Comment:
I would put these `enums` in a dedicated file (e.g., `GEMCalibEnums.h`).
See, e.g., `gembase/include/gem/base/utils/GEMEnums.h`